CareKit/CareKit/Synchronized View Controllers/TaskCategory/ViewControllers/OCKDetailedTaskCategoryViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
```

Comment:
nitpick: Remove the first `//` at the top of new files.

Reply:
Done